pandas/tests/io/excel/conftest.py

```diff
@@ -7,6 +7,9 @@
 
 @pytest.fixture
 def frame(float_frame):
+    """
+    Returns the first ten items in Fixture "float_frame"
```

Comment:
is Fixture supposed to be capitalized?  Needs a period at the end of this, but otherwise looks good.